pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -48,8 +59,8 @@ func NewCgroupManager(cs *cgroupSubsystems) CgroupManager {
 // Exists checks if all subsystem cgroups already exist
 func (m *cgroupManagerImpl) Exists(name string) bool {
```

Comment:
Exists should not return true unless the cgroup values are also updated... Imagine a scenario where kubelet restarts after creating cgroups but before updating cgroup values.